src/Persistence/Initialization/Skills/ReflectionEffectInitializer.cs

```diff
@@ -0,0 +1,60 @@
+﻿// <copyright file="ReflectionEffectInitializer.cs" company="MUnique">
+// Licensed under the MIT License. See LICENSE file in the project root for full license information.
+// </copyright>
+
+namespace MUnique.OpenMU.Persistence.Initialization.Skills;
+
+using MUnique.OpenMU.AttributeSystem;
+using MUnique.OpenMU.DataModel.Attributes;
+using MUnique.OpenMU.DataModel.Configuration;
+using MUnique.OpenMU.GameLogic.Attributes;
+
+/// <summary>
+/// Initializer for the reflection buff effect.
+/// </summary>
+public class ReflectionEffectInitializer : InitializerBase
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="ReflectionEffectInitializer"/> class.
+    /// </summary>
+    /// <param name="context">The context.</param>
+    /// <param name="gameConfiguration">The game configuration.</param>
+    public ReflectionEffectInitializer(IContext context, GameConfiguration gameConfiguration)
+        : base(context, gameConfiguration)
+    {
+    }
+
+    /// <inheritdoc/>
+    public override void Initialize()
+    {
+        var magicEffect = this.Context.CreateNew<MagicEffectDefinition>();
+        this.GameConfiguration.MagicEffects.Add(magicEffect);
+        magicEffect.Number = (byte)MagicEffectNumber.Reflection;
+        magicEffect.Name = "Reflection";
+        magicEffect.InformObservers = true;
+        magicEffect.SendDuration = false;
+        magicEffect.StopByDeath = true;
+        var powerUpDefinition = this.Context.CreateNew<PowerUpDefinition>();
+        magicEffect.PowerUpDefinitions.Add(powerUpDefinition);
+        powerUpDefinition.TargetAttribute = Stats.DamageReflection.GetPersistent(this.GameConfiguration);
+        magicEffect.Duration = this.Context.CreateNew<PowerUpDefinitionValue>();
+        magicEffect.Duration.ConstantValue.Value = 60;
+        var durationPerEnergy = this.Context.CreateNew<AttributeRelationship>();
+        durationPerEnergy.InputAttribute = Stats.TotalEnergy.GetPersistent(this.GameConfiguration);
+        durationPerEnergy.InputOperator = InputOperator.Multiply;
+        durationPerEnergy.InputOperand = 1f / 5f; // 5 energy adds 1 second duration
+        magicEffect.Duration.RelatedValues.Add(durationPerEnergy);
+
+        // one percent per 42 energy
+        var boostPerEnergy = this.Context.CreateNew<AttributeRelationship>();
+        boostPerEnergy.InputAttribute = Stats.TotalEnergy.GetPersistent(this.GameConfiguration);
+        boostPerEnergy.InputOperator = InputOperator.ExponentiateByAttribute;
+        boostPerEnergy.InputOperand = 1 - (0.01f / 42f);
```

Comment:
InputOperand = 0.01f / 40f

Reply:
Considering that it's 1% for every 42 energy points, is there a reason why it has to be divided by 40?
Or maybe the formula I put forward is not correct. I found it on a forum that claimed it was the "official" formula.

Reply:
Oh okay, I found this one: https://guidescroll.com/2011/08/mu-online-summoner-guide/

Reply:
I found this in the published sources:
https://forum.ragezone.com/threads/development-season-12-full-src-gs.1193275/page-4#post-9193953

![image](https://github.com/user-attachments/assets/0241839e-0f1b-4281-9775-a972127b614c)